### PR TITLE
(feat)chaos-charts: adding chaosengine yamls

### PR DESCRIPTION
**Pull request by aditya109**
Signed-off-by: aditya109 <adikid1996@gmail.com>

- added chaosengine for all the experiments

Solves : [1124](https://github.com/litmuschaos/litmus/issues/1124)

### CODE REVIEW

File: charts/coredns/coredns-pod-delete/engine.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-coredns
+  namespace: kube-system
+spec:
+  appinfo:
+    appns: kube-system
```
Comment (ispeakc0de) [inline charts/coredns/coredns-pod-delete/engine.yaml]: Can you please add single quotes also for  `appns`, `appkind` and `jobCleanUpPolicy` in all files

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

File: charts/openebs/openebs-pool-container-failure/engine.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  jobCleanUpPolicy: 'delete'
+  experiments:
+    - name: openebs-pool-container-failure
+      spec:
+        components:
+          env:
+            - name: APP_PVC
+              value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
+            - name: DEPLOY_TYPE
+              value: deployment
```
Comment (ispeakc0de) [inline charts/openebs/openebs-pool-container-failure/engine.yaml]: single quotes missing

File: charts/openebs/openebs-pool-pod-failure/engine.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    - name: openebs-pool-pod-failure
+      spec:
+        components:
+          env:
+            - name: FORCE
+              value: 'true'
+            - name: APP_PVC
+              value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
+            - name: DEPLOY_TYPE
+              value: deployment
```
Comment (ispeakc0de) [inline charts/openebs/openebs-pool-pod-failure/engine.yaml]: same as above

File: charts/openebs/openebs-target-container-failure/engine.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    - name: openebs-target-container-failure
+      spec:
+        components:
+          env:
+            - name: TARGET_CONTAINER
+              value: 'cstor-istgt'
+            - name: APP_PVC
+              value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
+            - name: DEPLOY_TYPE
+              value: deployment
```
Comment (ispeakc0de) [inline charts/openebs/openebs-target-container-failure/engine.yaml]: same as above

File: charts/openebs/openebs-target-network-delay/engine.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    - name: openebs-target-network-delay
+      spec:
+        components:
+          env:
+            - name: TARGET_CONTAINER
+              value: 'cstor-istgt'
+            - name: APP_PVC
+              value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
+            - name: DEPLOY_TYPE
+              value: deployment
```
Comment (ispeakc0de) [inline charts/openebs/openebs-target-network-delay/engine.yaml]: same as above

File: charts/openebs/openebs-target-network-loss/engine.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    - name: openebs-target-network-loss
+      spec:
+        components:
+          env:
+            - name: TARGET_CONTAINER
+              value: 'cstor-istgt'
+            - name: APP_PVC
+              value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
+            - name: DEPLOY_TYPE
+              value: deployment
```
Comment (ispeakc0de) [inline charts/openebs/openebs-target-network-loss/engine.yaml]: same as above

File: charts/openebs/openebs-target-pod-failure/engine.yaml [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    - name: openebs-target-pod-failure
+      spec:
+        components:
+          env:
+            - name: FORCE
+              value: 'true'
+            - name: APP_PVC
+              value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
+            - name: DEPLOY_TYPE
+              value: deployment
```
Comment (ispeakc0de) [inline charts/openebs/openebs-target-pod-failure/engine.yaml]: same as above

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

File: charts/generic/pod-network-corruption/engine.yaml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        components:
+          env:
+            - name: ANSIBLE_STDOUT_CALLBACK
+              value: 'default'
+              #Container name where chaos has to be injected
+            - name: TARGET_CONTAINER
+              value: 'nginx' 
+              #Network interface inside target container              
+            - name: NETWORK_INTERFACE
+              value: eth0
```
Comment (ispeakc0de) [inline charts/generic/pod-network-corruption/engine.yaml]: missing quotes

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

File: charts/generic/node-drain/engine.yaml [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    applabel: 'app=nginx'
+    appkind: 'deployment'
+  chaosServiceAccount: nginx-sa
+  monitoring: false
+  components:
+    runner:
+      image: 'litmuschaos/chaos-executor:1.0.0'
+      type: 'go'
+  # It can be delete/infra
+  jobCleanUpPolicy: 'delete'
```
Comment (amitbhatt818) [inline charts/generic/node-drain/engine.yaml:22]: replace `infra` to `retain`

File: charts/openebs/openebs-pool-container-failure/engine.yaml [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+    applabel: 'app=nginx'
+    appkind: 'deployment'
+  chaosServiceAccount: nginx-sa
+  monitoring: false
+  components:
+    runner:
+      image: 'litmuschaos/chaos-executor:1.0.0'
+      type: 'go'
+  # It can be delete/infra
+  jobCleanUpPolicy: 'delete'
```
Comment (amitbhatt818) [inline charts/openebs/openebs-pool-container-failure/engine.yaml:22]: same as above

File: charts/openebs/openebs-pool-pod-failure/engine.yaml [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+    appns: 'default'
+    applabel: 'app=nginx'
+    appkind: 'deployment'
+  chaosServiceAccount: nginx-sa
+  monitoring: false
+  components:
+    runner:
+      image: 'litmuschaos/chaos-executor:1.0.0'
+      type: 'go'
+  # It can be delete/infra
```
Comment (amitbhatt818) [inline charts/openebs/openebs-pool-pod-failure/engine.yaml]: Here also replace infra to retain

File: charts/openebs/openebs-target-container-failure/engine.yaml [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+    appns: 'default'
+    applabel: 'app=percona'
+    appkind: 'deployment'
+  chaosServiceAccount: nginx-sa
+  monitoring: false
+  components:
+    runner:
+      image: 'litmuschaos/chaos-executor:1.0.0'
+      type: 'go'
+  # It can be delete/infra
```
Comment (amitbhatt818) [inline charts/openebs/openebs-target-container-failure/engine.yaml]: same as above

File: charts/openebs/openebs-target-network-delay/engine.yaml [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+    appns: 'default'
+    applabel: 'app=nginx'
+    appkind: 'deployment'
+  chaosServiceAccount: nginx-sa
+  monitoring: false
+  components:
+    runner:
+      image: 'litmuschaos/chaos-executor:1.0.0'
+      type: 'go'
+  # It can be delete/infra
```
Comment (amitbhatt818) [inline charts/openebs/openebs-target-network-delay/engine.yaml]: same as above

File: charts/openebs/openebs-target-pod-failure/engine.yaml [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+    appns: 'default'
+    applabel: 'app=percona'
+    appkind: 'deployment'
+  chaosServiceAccount: nginx-sa
+  monitoring: false
+  components:
+    runner:
+      image: 'litmuschaos/chaos-executor:1.0.0'
+      type: 'go'
+  # It can be delete/infra
```
Comment (amitbhatt818) [inline charts/openebs/openebs-target-pod-failure/engine.yaml]: same as above

**Review by amitbhatt818 — CHANGES_REQUESTED** [review]

**Review by amitbhatt818 — APPROVED** [review]
/lgtm

**Review by ispeakc0de — APPROVED** [review]
/lgtm

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**